model/jpa/src/main/resources/META-INF/jpa-changelog-2.3.0.xml

```diff
@@ -36,6 +36,16 @@
          <addColumn tableName="IDENTITY_PROVIDER">
              <column name="PROVIDER_DISPLAY_NAME" type="VARCHAR(255)"></column>
          </addColumn>
+
+         <addColumn tableName="REALM">
+             <column name="SSO_MAX_LIFESPAN_REMEMBER_ME" type="INT" defaultValueNumeric="3600"/>
```

Comment:
should the default be 1 hour or 10 hours (as in the examples above e.g.: basicauthrealm.json)?